docs/Model/Response.md

```diff
@@ -5,6 +5,5 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **status** | **bool** |  | [optional]
-**result** | **object** |  | [optional]
```

Comment:
where is result now?

Reply:
removed recently https://github.com/qase-tms/specs/pull/84